charts/localstack/templates/deployment.yaml

```diff
@@ -17,6 +19,7 @@ spec:
       annotations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+        {{- include "localstack.annotations" . | nindent 8 }}
```

Comment:
I think you want to skip this actually... `podAnnotations` is generally treated different in K8S from the Deployment annotations. We already have the podAnnotations setting.

Also, this wouldn;t work right if the user did NOT set the podAnnotations, because line 19 won't be printed out...

Reply:
Fixed it.